src/components/address-book/ImportDialog/index.tsx

```diff
@@ -16,9 +16,18 @@ import css from './styles.module.css'
 import { trackEvent, ADDRESS_BOOK_EVENTS } from '@/services/analytics'
 import { abCsvReaderValidator, abOnUploadValidator } from './validation'
 import ErrorMessage from '@/components/tx/ErrorMessage'
+import { Errors, logError } from '@/services/exceptions'
 
 type AddressBookCSVRow = ['address', 'name', 'chainId']
 
+// https://react-papaparse.js.org/docs#errors
+type PapaparseErrorType = {
```

Comment:
This [can also have an (optional) `index` property](https://github.com/mholt/PapaParse/blob/master/papaparse.js#L1524) and [`row` can be `undefined`](https://github.com/mholt/PapaParse/blob/master/papaparse.js#L1378-L1380)

Reply:
Thanks for the observations. I've pushed an improved type